packages/builder/package.json

```diff
@@ -16,6 +15,7 @@
     "tooling",
     "task-runner"
   ],
+  "main": "build/index.js",
```

Comment:
You export a single function -- `mkAccounts` whereas `lib.ts` exoprts many things. Could you move `mkAccounts` to `lib.ts`?

Reply:
`internal/lib/lib.ts`?  - it doesn't export any function. It exports only `env`. This is not what we want to be imported by external applications.

Reply:
Then it would be best to remove or refactor it.

Reply:
No, because it's used in scripts and in the `console`.

Moreover there is another task to clean up the `lib.ts` and we even discussed yesterday.

Reply:
ok